File: A1/A1-019/A1-019.cpp
#include<iostream>
using namespace std;

int main()
{
    // This makes std::cin and std::cout faster
    ios_base::sync_with_stdio(false); cin.tie(0);

    // Input
    int a, b, c;
    cin >> a >> b >> c;
    // Output
    if(a == b && b == c) { cout << "all the same"; }
    else if(a != b && b != c && a != c) { cout << "all different"; }
    else { cout << "neither"; }
    return 0;
}

File: A1/A1-019/A1-019.py
# --------------------------------------------------
# File Name : A1-019.py
# Problem   : All the Same
# Author    : Worralop Srichainont
# Date      : 2026-02-08
# --------------------------------------------------

# Input
a = int(input())
b = int(input())
c = int(input())

# Output
if a == b == c:
    print("all the same")
elif a != b and b != c and a != c:
    print("all different")
else:
    print("neither")
